java/ql/src/Security/CWE/CWE-789/UnboundedAllocation.ql

```diff
@@ -0,0 +1,28 @@
+/**
+ * @name Unbounded allocation
```

Comment:
This query needs a description - I would expect tests to fail without one for a query in this folder! See https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-descriptions-description.

Maybe take inspiration from: https://help.semmle.com/wiki/display/JAVA/Uncontrolled+data+in+arithmetic+expression